pom.xml

```diff
@@ -1797,6 +1797,9 @@
 
                 <!-- tests should be updated -->
                 <exclude>**/AbstractJavadocCheckTest.class</exclude>
+
+                <!-- due to https://github.com/checkstyle/checkstyle/pull/13179 -->
+                <exclude>**/FullIdentTest.class</exclude>
```

Comment:
Please put this above `all bellow until` line of this tool exclude list.

Reply:
done